instaclustr/unit_test_utils.go

```diff
@@ -0,0 +1,45 @@
+package instaclustr
```

Comment:
Since this is all Kafka specific, could we rename the file to kakfa_unit_test_utils?

Reply:
OK